style/new_geisha/new_geisha_legacy.css

```diff
@@ -0,0 +1,611 @@
+/* Helper Classes */
```

Comment:
What's the role of this file vs `new_geisha.css`?

Reply:
It has all of the existing styles that go against the new design. So eventually these when we update the CSS of existing pages (like new Profile, new Travel, new VillageHQ) we'll be able to totally delete `new_geisha_legacy.css`.

So its basically a way to say, everything in `new_geisha.css` is good, everything in `new_geisha_legacy.css` needs replaced.

Reply:
Makes sense!